### ISSUE TITLE
Wrong scheme for compound query requests

### ISSUE THREAD

**Issue by LukasPietzschmann** [flowr-analysis/flowr#1029]
I was unsure if this should be tracked in #1007 or if a separate issue would be better, so feel free to close this one if #1007 is a better place for it.

---

As mentioned in the [doc](https://github.com/flowr-analysis/flowr/wiki/Query-API#compound-query), compound queries could look like the following:
```json
[
  {
    "type": "compound",
    "query": "call-context",
    "commonArguments": {
      "kind": "visualize",
      "subkind": "text"
    },
    "arguments": [
      {
        "callName": "^mean$"
      },
      {
        "callName": "^print$"
      }
    ]
  }
]
```
Note that we do not need to specify the type of all arguments, since it's already given in the `query` field. The scheme checked by the server however enforces the `type` field inside all arguments.

**Comment by EagleoutIce** [comment]
Yes, thank you, this is already fixed in the stage of [v2.1.0](https://github.com/flowr-analysis/flowr/pull/1018) which (automatically) updates the documentation accordingly and refines the schema generation! I hope i can merge it soon.

**Comment by LukasPietzschmann** [comment]
Ah, sorry! I did not check there.
